src/foam/dao/XMLDAO.java

```diff
@@ -13,17 +13,17 @@
         extends MapDAO
 {
 
-    private String fileName;
+    protected String fileName_;
 
-    public void setFileName(String filename) {
+    public void setFileName_(String filename) {
```

Comment:
The function is named setFileName() because it's public, only the internal var, which is protected should have the _.